plugins/base/src/test/kotlin/transformerBuilders/PageTransformerBuilderTest.kt

```diff
@@ -177,40 +177,6 @@ class PageTransformerBuilderTest : BaseAbstractTest() {
         }
     }
 
-    @Test
-    fun `should load script as defer if name ending in _deferred`() {
```

Comment:
I wrote this test a while ago, and at the time I couldn't find a good place for it, but I think having it in `ResourceLinksTest` is better now, so I moved it there.